### ISSUE TITLE
nightwatch-accessibility compatibility with nightwatch 2.x

### ISSUE THREAD

**Issue by Dossar** [ahmadnassri/node-nightwatch-accessibility#161]
Hello,

I noticed that https://github.com/ahmadnassri/node-nightwatch-accessibility/pull/160 was opened from dependabot to try upgrading nightwatch to the next major version. When I try using the current latest version of [nightwatch-accessibility](https://github.com/ahmadnassri/node-nightwatch-accessibility/releases/tag/v1.11.3) with nightwatch 2.x it fails with "aXe failed to execute" with no more information logged about what happened.

I do understand that error message comes from here:
https://github.com/ahmadnassri/node-nightwatch-accessibility/blob/0ffdd4140873b70a7c8a76b8cc6e62ee82217fdd/assertions/accessibility.js#L30

What steps are recommended when it comes to compatibility with nightwatch 2.x?

**Comment by ahmadnassri** [comment]
[v2.0.0](https://github.com/ahmadnassri/node-nightwatch-accessibility/releases/tag/v2.0.0)  has been released with fixes and compatibility with nightwatch v2
